tests/pages/course-admin/tester-version-page.js

```diff
@@ -0,0 +1,18 @@
+import { attribute, create, visitable } from 'ember-cli-page-object';
+
+export default create({
+  activateTesterVersionButton: {
+    scope: '[data-test-activate-tester-version-button]',
+  },
+
+  dataTestDeprovisionTestRunnersButton: {
```

Comment:
```suggestion
 deprovisionTestRunnersButton: {
```

@libmartinito this and the text (test runner has no test runners) is pretty careless btw, have to get better at attention to detail